estela-web/src/shared/notifications/index.tsx

```diff
@@ -10,6 +10,13 @@ export const authNotification = (): void => {
     });
 };
 
+export const apiCallFail = (): void => {
```

Comment:
@webtaken Is this resource being used?

Reply:
I deleted the notification, I didn't see it in any document.